### PR TITLE
Added L5/6 non-Martinotti SST

### LINKED ISSUE #948: [NTR] L5/6 non-Martinotti cell
Note - please check that the term does not already exist (check OLS: https://www.ebi.ac.uk/ols/ontologies/cl)

**Preferred term label:**
L5/6 Non-martinotti morphology Sst expressing GABAergic cortical interneuron (Mus musculus)

**Synonyms**
L5/6 Non-martinotti SST (Mus)

**Definition (free text, please give PubMed ID in format PMID:XXXXXX)**
A Sst GABAergic cortical interneuron of Sst subclass found in L5/L6 with non-Martionotti morphology. They have strongly facilitating excitatory inputs and moderately depressing outputs. They have powerful muscarinic depolarization, faster action potential than Martinotti and slightly slower action potential than PV basket cells. 
(PMID:27477017)

**Parent cell type term (check the hierarchy here https://www.ebi.ac.uk/ols/ontologies/cl):**
GABAergic interneuron

**What is the anatomical structure that the cell is a part of? Please check Uberon: https://www.ebi.ac.uk/ols/ontologies/uberon**
Cerebral Cortex

**Your nano-attribution (ORCID)**

**Any additional notes or concerns**

### PR DESCRIPTION
**Pull request by shawntanzk**
Fixes #948
awaiting some confirmation

### CODE REVIEW

**Comment by shawntanzk** [comment]
It is probable that all SST cells that are non-Martinotti in L5/6 are grouped here looking at dendrogram, however, do note that there this seems specific to L5/6 which I'm guessing is hard to precisely define as there is also a MET type which is L5 non-Martinotti. Similarly, there is L6 Th+ SST/PV cells which are non-Martinotti.

**Comment by shawntanzk** [comment]
also see closed PR https://github.com/obophenotype/cell-ontology/pull/949 for more discussion

**Review by patrick-lloyd-ray — APPROVED** [review]

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -28825,6 +28826,20 @@ AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> obo:CL_4023109 <ht
 AnnotationAssertion(rdfs:label obo:CL_4023109 "vasopressin-secreting magnocellular cell")
 EquivalentClasses(obo:CL_4023109 ObjectIntersectionOf(obo:CL_0011003 ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0030103)))
 
+# Class: obo:CL_4023118 (L5/6 non-Martinotti morphology sst expressing GABAergic cortical internueron (Mus musculus))
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:27477017"^^xsd:string) obo:IAO_0000115 obo:CL_4023118 "A sst GABAergic interneuron that has non-Martinotti morphology with a soma found in L5/6 of the cerebral cortex.")
```
Comment (shawntanzk) [inline src/ontology/cl-edit.owl:28831]: change non-Martinotti to be do not have Martinotti morphology

**Comment by shawntanzk** [comment]
Already looked at by @dosumis and made changes requested, will merge

**Pull request merged**